app/services/verify_partnership_service.rb

```diff
@@ -0,0 +1,22 @@
+class VerifyPartnershipService
+  attr_reader :client
+
+  def initialize(client)
+    @client = client
+  end
+
+  def call
+    verify_partnership
+  end
+
+  def api_partnership_data
+    { partners: %i[partner_company.com other_patner.com] }
```

Comment:
```suggestion
    { partners: %w[partner_company.com other_patner.com] }
```

Chuto que isso vá vir como um array de strings parece um pouco mais plausível, o que acha?

Reply:
Mudamos para o %w.